roadmap.md

```diff
@@ -0,0 +1,35 @@
+Roadmap Zombienet v2
+
+## Infra
+- Chaos testing, add examples and explore possibilities in `native` and `podman` provider
+- Add `docker` provider
+- Add `nomad` provider
+- Create [helm chart](https://helm.sh/docs/topics/charts/) to allow other use zombienet in k8s
+- Auth system to not use k8s users
+- Create GitHub Action and publish in marketplace (wip)
+
+## Internal teams
+- Add more teams (wip)
+
+## UI
+- Create UI to create `.feature` (`.zndls`) and `network` files.
+- Create script to translate from polkadot launch config to zombienet network file. ([wip](https://github.com/paritytech/zombienet/tree/nik-convert-polkadot-launch-config-to-zombienet))
+- Improve VSCode extension (grammar/snippets/syntax highlighting) ([repo](https://github.com/paritytech/zombienet-vscode-extension))
+- Add UI app (desktop) to run zombienet without the need of terminal 
+## Registry
+- Create decorators registry and allow override by paras (wip)
+- Explore how to get info from paras.
+
+## Functional tasks
+- Add subxt integration, allow to compile/run on the fly
+- Move parser to pest (wip)
```

Comment:
:heart: happy to collab on this :)

Reply:
Hey @drahnr, I started the work in this [draft pr](https://github.com/paritytech/zombienet/pull/382). The parser is already working and I need to finish the wiring between the parser crate and the test-runner. If you have time to review the pr would be awesome :)